pkg/ifreload/ifreload.go

```diff
@@ -0,0 +1,157 @@
+package ifreload
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	securityv1 "github.com/openshift/api/security/v1"
+	securityclient "github.com/openshift/client-go/security/clientset/versioned"
+	"github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/util/retry"
+
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+const (
+	kubeNamespace       = "openshift-azure-ifreload"
+	kubeServiceAccount  = "system:serviceaccount:" + kubeNamespace + ":default"
+	ifreloadImageFormat = "%s.azurecr.io/ifreload:0fa24761"
```

Comment:
hardcoded tag?

Reply:
yes, hardcoded

Reply:
ok, cool. I get the use now, but definatly worth some comment/readme about build/deploy cycle.
as i assumed we continuously build / push/ deploy (maybe 'cos i am doing the operator)